_includes/assets/js/indicatorModel.js

```diff
@@ -191,8 +191,13 @@ var indicatorModel = function (options) {
   }());
 
   var headlineColor = '777777';
-  var colors = ['7e984f', '8d73ca', 'aaa533', 'c65b8a', '4aac8d', 'c95f44'];
-
+  
+  // use custom colors if defined
+  this.colorSet = {{ site.graph_color_set | jsonify }};
+  this.numberOfColors = {{ site.graph_color_number | jsonify }};
+  this.customColors = {{ site.graph_color_list | jsonify }};
+  var colors = opensdg.chartColors(this.indicatorId, this.colorSet, this.numberOfColors, this.customColors);
```

Comment:
I believe you can use `this.shortIndicatorId` instead of `this.indicatorId` and you won't have to worry about the "indicator_" being in the front of it. Not a huge deal, just FYI.